src/pages/components/dropdown/accessibility.mdx

```diff
@@ -6,98 +6,129 @@ description:
+component.
 
 </PageDescription>
 
 <AnchorLinks>
-  <AnchorLink>Accessibility considerations</AnchorLink>
-  <AnchorLink>Resources</AnchorLink>
-  <AnchorLink>Accessibility testing</AnchorLink>
+
+<AnchorLink>What Carbon provides</AnchorLink>
+<AnchorLink>Development considerations</AnchorLink>
+
 </AnchorLinks>
 
-## Accessibility considerations
-
-1. Avoid very long option names to facilitate understandability and
-   perceivability.
-2. Do not use the same word or phrase at the beginning of a set of options.
-3. Avoid options that contain headings and interactive elements such as links,
-   buttons, or checkboxes.
-4. If the dropdown is a required field include the
-   [aria-required property](https://www.w3.org/TR/WCAG20-TECHS/ARIA2.html) and
-   indicate that it is a required field and use the validation message for input
-   errors.
-
-## Resources
-
-- [W3C WAI-ARIA Authoring Practices Listbox Design Pattern](https://www.w3.org/TR/wai-aria-practices-1.1/#Listbox)
-  covers the usage of ARIA names, state and roles, as well as the expected
-  keyboard interactions.
-- [IBM Accessibility Requirements](https://www.ibm.com/able/requirements/requirements/):
-  - [2.4.7 Focus Visible](https://www.ibm.com/able/requirements/requirements/#2_4_7)
-    (WCAG Success Criteria
-    [2.4.7](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html))
-  - [3.2.2 On Input](https://www.ibm.com/able/requirements/requirements/#3_2_2)
-    (WCAG Success Criteria
-    [3.2.2](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/consistent-behavior-unpredictable-change.html))
-  - [4.1.2 Name, Role, Value](https://www.ibm.com/able/requirements/requirements/#4_1_2)
-    (WCAG Success Criteria
-    [4.1.2](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value))
-
-## Accessibility testing
-
-Automated, manual and screen reader accessibility verification test has been
-performed on the dropdown React Carbon Component.
-[WCAG 2.1 Level A and AA success criteria](https://www.w3.org/TR/WCAG21/) issues
-have been identified and the list of open accessibility violations is available
-in the Carbon Component GitHub repository.
-
-### Automated test
+## What Carbon provides
+
+Carbon bakes keyboard operation into its components, improving the experience of
+blind users and others who operate via the keyboard. Carbon incorporates many
+other accessibility considerations, some of which are described below.
+
+### Keyboard interactions
+
+A dropdown component and its variants multiselect and combo box are reached by
+`Tab`, with navigation of the options by `Up` and `Down` arrow keys. However,
+the keys for opening the component and selecting its options are different for
+each variant.
+
+<Row>
+<Column colLg={8}>
+
+![Tab to dropdown then operate with arrow keys](images/dropdown-accessibility-1.png)
+
+<Caption>
+
+Dropdowns are in the tab order and operable by Up and Down arrow keys, once
+opened.
+
+</Caption>
```

Comment:
the type format looks correct here but it doesn't look like it's in the caption on the site. Wondering if there should be no space between the Caption and the text.

Reply:
I've pushed that change. Just waiting to check the deployment before resolving this conversation.